test/ex_oauth2_provider/oauth2/authorization/strategy/code_test.exs

```diff
@@ -134,12 +140,20 @@ defmodule ExOauth2Provider.Authorization.CodeTest do
       %{resource_owner: resource_owner, application: application}
     end
 
+    test "generates grant with no scope passed", %{resource_owner: resource_owner} do
```

Comment:
Same here, the describe is a specific test case. I would put this test outside the describe context.